scripts/build

```diff
@@ -171,7 +171,6 @@ if [ "${BUILD_WITH_DEBUG}" != "yes" ] && flag_enabled "strip" "yes"; then
   TARGET_MESON_OPTS+=" -Dstrip=true"
 fi
 
-
```

Comment:
Double line breaks are done for grouping/dividing. In this example, TARGET from HOST options.

Reply:
> Double line breaks are done for grouping/dividing.

I really do not like the idea of grouping with wrong style formats. I guess I add a comment instead ?

Reply:
I don't see how it's a wrong style. What rule is the script enforcing and can it be disabled? Can it show just changes for applying that rule? The spot I commented is just an example and this double line break is done in other places.

Reply:
@antonlacon looked through all changes and we have just 3 cases where this is changed and in all cases the double line breaks are followed by a comment that not really separates anything 
from my perspective that is a non issue as the extra line break makes nothing more readable or similar
![image](https://github.com/user-attachments/assets/2b3b9734-0923-4822-ab14-d2ebbc66e276)

![image](https://github.com/user-attachments/assets/4d6dd072-3bdf-419a-982b-e32b3f18d6a1)